src/legacy/state/price/updater.ts

```diff
@@ -275,5 +261,18 @@ export default function FeesUpdater(): null {
     sellTokenAddressInvalid,
   ])
 
+  /**
+   * Periodically fetch quotes
+   */
+  useInterval({
+    callback: refetchQuoteIfRequired,
+    name: 'FeesUpdater',
+    delay: REFETCH_CHECK_INTERVAL,
+    triggerEagerly: true,
```

Comment:
Should this be `false` given that there's an `useEffect` below triggering when the callback changes?

```suggestion
    triggerEagerly: false,
```